app/controllers/WvypStreamJava.java

```diff
@@ -1,45 +0,0 @@
-package controllers;
-
-import data.ServiceClientJ;
-import play.twirl.api.Html;
-import play.libs.F;
-import play.mvc.Controller;
-import play.mvc.Result;
-import scala.Function1;
-import ui.HtmlStream;
-import ui.Pagelet;
-
-import java.util.Arrays;
-
-public class WvypStreamJava extends Controller
```

Comment:
Same question about why delete? This is a Java example to show that these techniques work in Java as well as Scala.